lib/ansible/modules/web_infrastructure/ansible_tower/tower_job_template.py

```diff
@@ -176,6 +182,8 @@ def update_fields(p):
         'ask_limit': 'ask_limit_on_launch',
         'ask_tags': 'ask_tags_on_launch',
         'ask_job_type': 'ask_job_type_on_launch',
+        'ask_inventory': 'ask_inventory_on_launch',
+        'ask_credential': 'ask_credential_on_launch',
```

Comment:
This becomes more complicated with the latest AWX version, but the field_map will have to be refactored in general.